6-AdvancedScenarios/4-sign-in-hybrid/App/controllers/authController.js

```diff
@@ -24,13 +24,14 @@ exports.loginUser = async (req, res, next) => {
  * to be redeemed by your single-page application.
  */
 exports.handleRedirectWithCode = (req, res, next) => {
+  const port = process.env.PORT || 5000;
```

Comment:
Can we declare this at the top instead of here please?

Reply:
Done